android/assets/jsons/translations/template.properties

```diff
@@ -1151,4 +1155,4 @@ in all cities with a garrison =
 
 Only available after [] turns = 
 This Unit upgrades for free = 
-
+[stats] when a city adopts this religion for the first time = 
```

Comment:
See, doesn't match - and aren't uniques picked up automaniacally?

Reply:
I implemented two uniques: one with the part between brackets and one without. The one with brackets is in a json so is automatically picked up, but the one without isn't so it is added here manually :)